examples/readme.md

```diff
@@ -10,7 +10,7 @@ There are also a few audio files to test with in the `static/` folder.
 Prerequisite
 ------------
 
-* IBM Watson Speech to Text service credentials - see http://www.ibm.com/watson/developercloud/doc/getting_started/gs-credentials.shtml
+* IBM Watson Speech to Text service credentials - see [Service credentials for Watson services](https://www.ibm.com/watson/developercloud/doc/common/getting-started-credentials.html)
```

Comment:
We can probably shorten the text here and just make the entire thing a link. But meh.